### PR TITLE
Logged msg when fetching bug is impossible

### LINKED ISSUE #25: Log exceptions while connecting to API
Get bug data mask all exceptions from such connection. A warning message can be logged so users can be aware of connection problem.

### PR DESCRIPTION
**Pull request by renzon**
close #25

Messages extracted from robottelo

### CODE REVIEW

File: robozilla/bz.py [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-import os
 import logging
 from collections import defaultdict
+import os
+from xml.parsers.expat import ExpatError, ErrorString
 
 import bugzilla
-
-# to do handle errors
+import fnmatch
```
Comment (rochacbruno) [inline robozilla/bz.py]: @renzon  where is fnmatch being used?

Comment (rochacbruno) [inline robozilla/bz.py]: Ahh ok, my mistake, I saw a green line and thought you added this :)

File: robozilla/bz.py [hunk 2]
```diff
@@ -160,9 +158,24 @@ def get_bug_data(self, bug_id):
...
+                logger.warning(
+                    'Could not fetch bug. Error: {0}'.format(error.faultString)
+                )
+            except ExpatError as error:
+                logger.warning(
+                    'Could not interpret bug. Error: {0}'.format(
+                        ErrorString(error.code)
+                    )
+                )
+            except (TypeError, ValueError):  # pragma: no cover
```
Comment (ldjebran) [inline robozilla/bz.py]: @renzon , think we should keep this raise (by putting  raise after logger.warning), think we need to we need to raise this exception.
@rochacbruno what do you think?

Comment (renzon) [inline robozilla/bz.py]: @ldjebran to be honest I copied the Exceptions handling from robotello. So I can move this portion back there.

**Pull request merged**